resolwe_bio/processes/support_processors/mutations_table.py

```diff
@@ -498,3 +536,62 @@ def run(self, inputs, outputs):
         outputs.genes = genes
         outputs.species = inputs.variants.output.species
         outputs.build = inputs.variants.output.build
+
+        # Store the reported variants in the Variant Database
+        data_source = "RNA-Seq Variant Calling Pipeline"
```

Comment:
Should this be an advanced parameter with the default value "RNA-Seq Variant Calling Pipeline" since any VCF file annotated with snpeff can be used as an input to this process?

Reply:
Added as an (advanced) input field, thanks!